pkg/utils/util_test.go

```diff
@@ -20,11 +20,101 @@ func TestContains(t *testing.T) {
 	})
 
 	t.Run("does not contain", func(t *testing.T) {
-
 		// when
 		ok := Contains(content, "4")
 
 		// then
 		assert.False(t, ok)
 	})
 }
+
+func TestGetToolchainClusterName(t *testing.T) {
+	type Params struct {
+		ServerAPI   string
+		ClusterType string
+		Ordinal     int
+	}
+	for expectedClusterName, params := range map[string]Params{
```

Comment:
btw, I don't see  any cases for host
and let's add something that is similar to what we have in prod:
```suggestion
	for expectedClusterName, params := range map[string]Params{
		"host-sandbox.x7a5.a2.openshiftapps.com": {
			ClusterType: "host",
			ServerAPI:   "https://api.sandbox.x7a5.a2.openshiftapps.com:6443",
			Ordinal:     0,
		},
		"member-sandbox-m2.ab8k.b3.openshiftapps.com": {
			ClusterType: "member",
			ServerAPI:   "https://api.sandbox-m2.ab8k.b3.openshiftapps.com:6443",
			Ordinal:     0,
		},
```